magpie-run

```diff
@@ -1042,6 +1042,159 @@ else
 fi
 
 #
+# Setup Phoenix
+#
+
+if [ "${PHOENIX_SETUP}" == "yes" ] && [ "${prior_setup_successful}" == "true" ]
+then
+    cd ${PHOENIX_HOME}
+
+    if [ ${PHOENIX_MODE} == "server" ]
+    then
+        # Make variables unspecified for launching
+	Magpie_make_all_local_dirs_unspecified
+
+	echo "Starting Phoenix Queryserver"
+        ${MAGPIE_SCRIPTS_HOME}/bin/magpie-phoenix.sh start 
+	
+        # My rough estimate for setup time is 30 seconds
+        sleepwait=30
+
+	echo "Waiting ${sleepwait} seconds to allow Phoenix daemons to setup"
+	sleep ${sleepwait}
+	totalsleepwait=`expr ${totalsleepwait} + ${sleepwait}`
+    fi
+
+    echo "*******************************************************"
+    echo "*"
+    echo "* Phoenix Information"
+    echo "*"
+    echo "* To access phoenix directly, you'll want to:"
+    echo "*   ${MAGPIE_REMOTE_CMD:-ssh}${MAGPIE_REMOTE_CMD_OPTS:+" "}${MAGPIE_REMOTE_CMD_OPTS} ${HBASE_MASTER_NODE}"
+    if echo $MAGPIE_SHELL | grep -q csh
+    then
+	echo "*   setenv HBASE_CONF_DIR \"${HBASE_CONF_DIR}\""
+    else
+	echo "*   export HBASE_CONF_DIR=\"${HBASE_CONF_DIR}\""
+    fi
+    echo "*   cd $PHOENIX_HOME"
+    echo "*"
+    echo "* Then you can do as you please.  To interact with Phoenix you can use:"
+    echo "*" 
+    echo "*   ${phoenixcmdprefix}/sqlline.py"
+
+    echo "*" 
+    echo "* Some additional environment variables you may sometimes wish to set"
+    echo "*" 
+    if echo $MAGPIE_SHELL | grep -q csh
+    then
+	echo "*   setenv JAVA_HOME \"${JAVA_HOME}\""
+	echo "*   setenv HBASE_HOME \"${HBASE_HOME}\""
+        echo "*   setenv PHOENIX_HOME=\"${PHOENIX_HOME}\""
+    else
+	echo "*   export JAVA_HOME=\"${JAVA_HOME}\""
+	echo "*   export HBASE_HOME=\"${HBASE_HOME}\""
+        echo "*   export PHOENIX_HOME=\"${PHOENIX_HOME}\""
+    fi
+
+    if [ "${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}X" != "X" ]
+    then
+	echo "*"
+	echo "* If running interactively, sourcing"
+	echo "*"
+	echo "* ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}"
+	echo "*"
+	echo "* will set most common environment variables for your job."
+    fi
+    echo "*" 
+    echo "*******************************************************"
+
+    if [ "${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}X" != "X" ]
+    then
+	if echo $MAGPIE_SHELL | grep -q csh
+	then
+	    echo "setenv HBASE_CONF_DIR \"${PHOENIX_CONF_DIR}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "setenv HBASE_LOG_DIR \"${PHOENIX_LOG_DIR}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+            echo "setenv PHOENIX_HOME=\"${PHOENIX_HOME}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	else
+	    echo "export HBASE_CONF_DIR=\"${PHOENIX_CONF_DIR}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "export HBASE_LOG_DIR=\"${PHOENIX_LOG_DIR}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+            echo "export PHOENIX_HOME=\"${PHOENIX_HOME}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	fi
```

Comment:
Perhaps this calls for another patch to Phoenix to look for PHOENIX_CONF_DIR over HBASE's? I'm not really sure what to do there since in general I don't think anyone would configure Phoenix outside of HBase. I just thought it would be nice to have the configs and logs all separate.